mongo/description/selector_test.go

```diff
@@ -368,7 +367,6 @@ func BenchmarkSelector_Sharded(b *testing.B) {
 func TestSelector_Single(t *testing.T) {
 	t.Parallel()
 
-	require := require.New(t)
```

Comment:
I did not include [this functionality](https://github.com/stretchr/testify/blob/master/require/forward_requirements.go#L10) (creating an `Assertions` struct and calling assertions as methods) in our copy of `testify`, as I found it redundant.